projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php

```diff
@@ -327,6 +327,15 @@ protected function update() {
 				continue;
 			}
 
+			// Rely on WP_Automatic_Updater class to check if a plugin item should be updated.
+			if ( ! class_exists( 'WP_Automatic_Updater' ) ) {
```

Comment:
I believe we should move this outside of the `foreach` loop.

If you look at line 314, there's already a:
`include_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php';`

Reply:
Thank you for catching that! https://github.com/Automattic/jetpack/pull/18707/commits/40220fd7e447d8d00e3e6b12c9e880731b889701